extensions/amp-story-interactive/0.1/amp-story-interactive-slider.css

```diff
@@ -116,6 +119,17 @@
   height: var(--thumb-size) !important;
   background: var(--interactive-accent-color) !important;
   border-radius: 50% !important;
+  position: relative;
+}
+
+.i-amphtml-story-interactive-slider-container.show
+  .i-amphtml-story-interactive-slider-input::-webkit-slider-thumb {
+  -webkit-appearance: none !important;
+  transform: rotate(45deg) !important;
+  box-shadow: 0em 0em 0.5em rgba(0, 0, 0, 0.25) !important;
+  z-index: 3 !important;
+  border-radius: 0% 50% 50% 50% !important;
```

Comment:
You can just update the border radius that you want with `border-bottom-left-radius: 0` for instance